NimiqClient/NimiqClient.csproj

```diff
@@ -0,0 +1,21 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.1</TargetFramework>
+    <PackageVersion>0.0.4</PackageVersion>
+    <Authors>Nimiq Community</Authors>
+    <Copyright>Copyright (c) 2020 Nimiq Community</Copyright>
+    <Owners>Nimiq Community</Owners>
+    <PackageProjectUrl>https://github.com/rraallvv/NimiqClientSharp</PackageProjectUrl>
+    <Summary>https://github.com/rraallvv/NimiqClientSharp/blob/master/README.md</Summary>
+    <PackageTags>Nimiq, JSON-RPC Client</PackageTags>

```

Comment:
If these files are needed, then the URLs should be updated.

Reply:
It's been [fixed here](https://github.com/rraallvv/csharp-client/commit/03108cca7d1cd868192c0e1b205d0f8107655cb5)